session/pessimistic_test.go

```diff
@@ -582,7 +562,11 @@ func (s *testPessimisticSuite) TestWaitLockKill(c *C) {
 	_, err := tk2.Exec("update test_kill set c = c + 1 where id = 1")
 	wg.Done()
 	c.Assert(err, NotNil)
-	c.Assert(terror.ErrorEqual(err, tikv.ErrQueryInterrupted), IsTrue)
+	if *withTiKV {
+		// Test not stable under TiKV
```

Comment:
Do you know the reason?

Reply:
Fixed.
The wait time is not long enough, so both `ErrQueryInterrupted` and `ErrLockWaitTimeout` are triggered. We check `ErrLockWaitTimeout` before `ErrQueryInterrupted`
https://github.com/pingcap/tidb/blob/master/store/tikv/2pc.go#L767